packages/@ourworldindata/utils/src/owidTypes.ts

```diff
@@ -690,6 +691,76 @@ export type EnrichedBlockSDGGrid = {
     items: EnrichedSDGGridItem[]
 } & EnrichedBlockWithParseErrors
 
+export type RawBlockStickyRightContainer = {
+    type: "sticky-right"
+    value: {
+        left: OwidRawArticleBlock[]
```

Comment:
This is typed a bit optimistically - we don't really know if the user really supplied a left and right value